client/app/views/duplicates/index.coffee

```diff
@@ -32,6 +32,10 @@ module.exports = class DuplicatesView extends Mn.CompositeView
     collectionEvents:
         'contacts:merge': 'render'
 
+    render: ->
+        super
+        $($('nav a').get(1)).attr 'aria-busy', false
```

Comment:
I suggest we use the `onRender` builtin method rather than superseed the native render.

We can call an `app.channel.trigger 'duplicates:render'` event on the global bus.